charts/qat/values.yaml

```diff
@@ -0,0 +1,16 @@
+qat:
+  metadata:
+    name: qatdeviceplugin-sample
+    annotations:
+      container.apparmor.security.beta.kubernetes.io/intel-qat-plugin: unconfined
```

Comment:
I think this does not need to be enabled by default since it's only needed on some Ubuntu versions.

Reply:
should I delete annotations?

Reply:
yes from the default deployment